hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/s3guard/S3GuardTool.java

```diff
@@ -1544,6 +1544,7 @@ private void vprintln(PrintStream out, String format, Object...
    */
   static class Fsck extends S3GuardTool {
     public static final String CHECK_FLAG = "check";
+    public static final String DDB_MS_CONSISTENCY_FLAG = "ddbIC";
```

Comment:
I don't like the name of this flag; Case-sensitive and not that obvious. Maybe `internal`?

Reply:
ok, I agree. changed that.